test/mobile/custom_build/CMakeLists.txt

```diff
@@ -13,12 +13,16 @@ target_include_directories(Predictor PUBLIC ${TORCH_INCLUDE_DIRS})
 
 find_package(Threads REQUIRED)
 
+# Avoid linker flags on Mac
+if(${APPLE})
+  set(LINKER_OPTIONS "")
+else()
+  set(LINKER_OPTIONS "-Wl,-s -Wl,--gc-sections -Wl,--whole-archive -Wl,--no-whole-archive")
```

Comment:
This is wrong. `-Wl,--whole-archive` linker flag positioning matters: you have to enclose the objects you want to whole archive with the whole-archive/no-whole-archive; moving them here turns it into a noop